packages/mattermost-redux/src/reducers/entities/apps.ts

```diff
@@ -13,6 +13,9 @@ import {validateBindings} from 'mattermost-redux/utils/apps';
 
 export function bindings(state: AppBinding[] = [], action: GenericAction): AppBinding[] {
     switch (action.type) {
+    case AppsTypes.FAILED_TO_FETCH_APP_BINDINGS: {
+        return [];
+    }
```

Comment:
I agree with you. We should invalidate no matter the error. I am trying to fabricate an scenario where this would be a problem, but I cannot think of any real enough. If we get to any problem in the future, the fix is simple.